dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -34,7 +34,7 @@ log.directory.desc=You can use this attribute to set a directory for all log fil
 message.file.name=Message file name
 message.file.name.desc=Name of the file to which message output is written relative to the configured log directory. The default value is messages.log. This file always exists and contains INFO and other (AUDIT, WARNING, ERROR, FAILURE) messages, in addition to System.out and System.err. This log also contains time stamps and the issuing thread ID. If the log file is rolled over, the names of earlier log files have the format messages_timestamp.log. Avoid trouble: Use the com.ibm.ws.logging.message.file.name property, except in cases where you are trying to change the configuration dynamically after startup.
 
-stackJoin=Format stack traces as a single event in logs.
+stackJoin=Format stack traces as a single event in logs
```

Comment:
I think this does get a period. It reads kind of like a fragment, but it's a complete imperative sentence. If we're going to use a period after the desc at line 38, we should use one here too

Reply:
@dmuelle this attribute/label isn't allowed a period according to the `ant_schema` validator unittests :(